things/api/things/http/transport.go

```diff
@@ -172,6 +172,76 @@ func MakeHandler(tracer opentracing.Tracer, svc things.Service) http.Handler {
 		opts...,
 	))
 
+	r.Get("/things/:memberID/groups", kithttp.NewServer(
```

Comment:
`thingID`

Reply:
reason why it is `memberID` is because the methods that server this endpoint could be used for other entities that we want to group

Reply:
Until we decide let's have something consistent

Reply:
we need to decide first

Reply:
What I meant is that if we finally don't group you will have to remember and modify this. If we group this will moved from here in any case.